fabric-resource-conditions-api-v1/src/main/java/net/fabricmc/fabric/mixin/resource/conditions/DataPackContentsMixin.java

```diff
@@ -39,4 +46,12 @@ public class DataPackContentsMixin {
 	public void hookRefresh(DynamicRegistryManager dynamicRegistryManager, CallbackInfo ci) {
 		ResourceConditionsImpl.clearTags();
```

Comment:
Could you also clear the feature set ThreadLocal ? (And move the mixin's javadoc to the `hookRefresh` function.

Reply:
There should be no need. Feature flags are immutable during server lifetime, and server loading always starts with reload method.